yellow-paper/docs/state/index.md

```diff
@@ -4,12 +4,43 @@ sidebar_position: 10
 ---
 
 # State
+The global state is the set of data that makes up Aztec - it is persisted and only updates when new blocks are added to the chain. The state consists of multiple different categories of data with varying requirements. Common for all of the categories is that they need strong integrity guarantees and efficient membership proofs. Like for most other blockchains, this can be enforced by structuring the data as leafs in Merkle trees. However, unlike most other blockchains, our contract state cannot use a Merkle tree as a key-value store for each contracts data. The reason being that we have both private and public state. While public state could be stored in a key-value tree, private state cannot - as doing so would leak information whenever the private state is updated, even if encrypted.
```

Comment:
```suggestion

The global state is the set of data that makes up Aztec - it is persistent and only updates when new blocks are added to the chain. 

The state consists of multiple different categories of data with varying requirements. What all of the categories have in common is that they need strong integrity guarantees and efficient membership proofs. Like most other blockchains, this can be enforced by structuring the data as leafs in Merkle trees. 

However, unlike most other blockchains, our contract state cannot use a Merkle tree as a key-value store for each contracts data. The reason for this is that we have both private and public state; while public state could be stored in a key-value tree, private state cannot, as doing so would leak information whenever the private state is updated, even if encrypted.

```